test/blackbox-tests/test-cases/dune-cache/cache-man.t

```diff
@@ -0,0 +1,181 @@
+Here we observe the documentation for the dune cache commands.
```

Comment:
At Jane Street, we have automation for autogenerating the output of `--help` across all supported commands/actions, for committing it into the repository. I wonder if we can add similar functionality to Dune, to avoid writing these tests by hand.

Reply:
We could easily setup some promotion rules for man page generation I believe. I can leave this to another PR however and remove this test once we do that.

Reply:
Sure, this doesn't need to happen in this PR.